frontend/public/style/_forms.scss

```diff
@@ -6,3 +6,41 @@ label.co-required:after {
   content: '*';
   padding-left: 3px;
 }
+
+.form-right-padding {
```

Comment:
I feel like most of these styles should be unnecessary if we just use Bootstrap form styles. I'm not sure we need any custom classes for this form.

cc @rhamilto

Reply:
The text runs together when I do things in-line horizontally.

Reply:
> The text runs together when I do things in-line horizontally.

Can you give an example? I'd guess you're missing some of the Bootstrap classes if this is the case.

Reply:
My radio buttons for the access mode run together- the text touches the next radio circle.  Bootstrap says it has "form-check-inline" for horizontal check/radio options, but it does not seem to be in our code base.
Here is what it looks like without the padding:
![radio-smashed-together](https://user-images.githubusercontent.com/18728857/47302823-afe14700-d5df-11e8-9803-634110501e29.png)